plugins/inputs/opcua/README.md

```diff
@@ -1,73 +1,73 @@
-# Telegraf Input Plugin: opcua_client
+# OPC UA Client Input Plugin
 
-The opcua_client plugin retrieves data from OPCUA slave devices
+The `opcua` plugin retrieves data from OPC UA client devices.
+
+Telegraf minimum version: Telegraf 1.16
+Plugin minimum tested version: 1.16
 
 ### Configuration:
 
 ```toml
-
-# ## Connection Configuration
-#  ##
-#  ## The plugin supports connections to PLCs via OPCUA
-#  ##
-#  ## Device name
-name = "opcua_rocks"
+[[inputs.opcua]]
+## This plugin supports connections to PLCs via OPC UA.
+##
+## Device name
+name = "localhost"
 #
-#  # OPC UA Endpoint URL
-endpoint = "opc.tcp://opcua.rocks:4840"
+## OPC UA Endpoint URL
+endpoint = "opc.tcp://localhost:4840"
 #
-#  ## Read Timeout
-#  ## add an arbitrary timeout (seconds) to demonstrate how to stop a subscription
-#  ## with a context.
+## Read Timeout. Add an arbitrary timeout (seconds) to demonstrate how to stop a subscription with a context.
 timeout = 30
 #
-#  # Time Inteval, default = 10s
-time_interval = "5s"
+## Time Interval (Default = 10s)
+time_interval = "10s"
 #
-#  # Security policy: None, Basic128Rsa15, Basic256, Basic256Sha256. Default: auto
+## Security policy: None, Basic128Rsa15, Basic256, Basic256Sha256.
 security_policy = "None"
 #
-#  # Security mode: None, Sign, SignAndEncrypt. Default: auto
+## Security mode: None, Sign, SignAndEncrypt.
 security_mode = "None"
 #
-#  # Path to cert.pem. Required for security mode/policy != None. If cert path is not supplied, self-signed cert and key will be generated.
-#  # certificate = "/etc/telegraf/cert.pem"
+## Path to cert.pem. Required for security mode/policy != None. If cert path is not supplied, self-signed cert and key will be generated.
+## certificate = "/etc/telegraf/cert.pem"
 #
-#  # Path to private key.pem. Required for security mode/policy != None. If key path is not supplied, self-signed cert and key will be generated.
-#  # private_key = "/etc/telegraf/key.pem"
+## Path to private key.pem. Required for security mode/policy != None. If key path is not supplied, self-signed cert and key will be generated.
+## private_key = "/etc/telegraf/key.pem"
 #
-#  # To authenticate using a specific ID, select chosen method from 'Certificate' or 'UserName'. Else use 'Anonymous.' Defaults to 'Anonymous' if not provided.
-#  # auth_method = "Anonymous"
+## To authenticate using a specific ID, select chosen method from 'Certificate' or 'UserName'. Else use 'Anonymous.' Defaults to 'Anonymous' if not provided.
```

Comment:
What does "else use" mean here? That you should pick anonymous if you're not using a specific ID to authenticate?

Reply:
yeah so maybe just or else use anonymous if you're not using `auth_method = "Anonymous"` or `auth_method = "Certificate"`